monitor/grafana/zeebe.json

```diff
@@ -7748,7 +7748,7 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "editorMode": "code",
-              "expr": "histogram_quantile(0.50, sum(rate(zeebe_messaging_request_response_latency_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\",pod=~\"$pod\", partition=~\"$partition\"}[$__rate_interval])) by (topic, le))",
+              "expr": "histogram_quantile(0.50, sum(rate(zeebe_messaging_request_response_latency_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\",pod=~\"$pod\", partition=~\"$partition\"}[$__rate_interval]) or rate(zeebe_messaging_request_response_latency_seconds_bucket{cluster=~\"$cluster\", namespace=~\"$namespace\",pod=~\"$pod\", partition=~\"$partition\"}[$__rate_interval])) by (topic, le))",
```

Comment:
:wrench: Does this show things properly? I ran into an issue with the union of series and heatmap data earlier, though I was not using histogram_quantile, so maybe that works better.

Reply:
They look fine to me, similar to other benchmarks 
![image](https://github.com/user-attachments/assets/4280d0f5-7012-4a63-ae41-d925f2b5678c)